### PR TITLE
JS: Fix missing 'require' statements

### PR DESCRIPTION
**Pull request by dtymon**
This is a simple patch to fix some missing 'require' statements in the JS runtime.

### CODE REVIEW

File: runtime/JavaScript/src/antlr4/atn/PredictionMode.js [hunk 1]
```diff
@@ -38,6 +38,9 @@ var BitSet = require('./../Utils').BitSet;
 var AltDict = require('./../Utils').AltDict;
 var ATN = require('./ATN').ATN;
 var RuleStopState = require('./ATNState').RuleStopState;
+var ATNConfigSet = require('./ATNConfigSet');
```
Comment (ericvergnaud) [inline runtime/JavaScript/src/antlr4/atn/PredictionMode.js]: missing .ATNConfigSet?

File: runtime/JavaScript/src/antlr4/atn/PredictionMode.js [hunk 2]
```diff
@@ -38,6 +38,9 @@ var BitSet = require('./../Utils').BitSet;
...
+var ATNConfig = require('./ATNConfig');
```
Comment (ericvergnaud) [inline runtime/JavaScript/src/antlr4/atn/PredictionMode.js]: Missing .ATNConfig?

**Comment by parrt** [comment]
@ericvergnaud do you bless this change?

**Comment by parrt** [comment]
I cherrypicked contrib for next PR.

**Comment by ericvergnaud** [comment]
blessed

**Comment by parrt** [comment]
thanks Eric!

**Pull request merged**